publish/publish.proj

```diff
@@ -30,7 +30,21 @@
              BuildInParallel="true" />
   </Target>
 
+  <Target Name="EnsureUniqueItemsToUpload">
+    <ItemGroup>
+      <DedupItems Include="@(ItemsToPublish->Distinct())" />
+    </ItemGroup>
+    <PropertyGroup>
+      <CountOld>@(ItemsToPublish->Count())</CountOld>
+      <CountNew>@(DedupItems->Count())</CountNew>
+      <_HasDups Condition="'$(CountOld)' != '$(CountNew)'">true</_HasDups>
+    </PropertyGroup>
+    <Message Importance="High" Condition="'$(_HasDups)' != ''" Text="ItemsToPublish: '@(ItemsToPublish)'"/>
```

Comment:
I guess, if we're just displaying all of the items, and not just the duplicates, then it could be an obnoxious amount of red.  It's probably fine as-is.